```python
    training_inputs = [np.reshape(x, (784, 1)) for x in tr_d[0]]
    training_results = [vectorized_result(y) for y in tr_d[1]]
    training_data = zip(training_inputs, training_results)

    print_shape(list(training_data)[0:1])
```

```sh
% python mnist_loader_3.py

[5 0 4 ... 8 4 8]
(50000,)
Traceback (most recent call last):
  File "mnist_loader_3.py", line 61, in <module>
    td, vd, td1 = load_data_wrapper()
  File "mnist_loader_3.py", line 42, in load_data_wrapper
    print_shape(list(training_data)[0:1])
  File "mnist_loader_3.py", line 19, in print_shape
    arr = np.array(array)
ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 2 dimensions. The detected shape was (1, 2) + inhomogeneous part.
```
